sktime/transformations/tests/test_ytox.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import pandas as pd
+import numpy as np
+from sktime.datasets import load_airline
+from sktime.transformations.compose import YtoX
+from sktime.transformations.series.exponent import ExponentTransformer
+from sktime.transformations.series.boxcox import BoxCoxTransformer
+
+
+@pytest.fixture
+def y_data():
+    """Fixture to create a sample dataset."""
+    y = load_airline()
+    return y
+
+
+def get_test_params():
+    """Provide test parameters for the YtoX class."""
+    # Return instances with different parameters
+    return [
+        {"ytox": YtoX(subset_index=False), "description": "YtoX without subset_index"},
```

Comment:
Add only two parameter sets, also I don't see `description` as parameter in` __init__` of class YtoX, it needs to removed.

Reply:
it was a description for what parameters I wrote as reminder for me, I have removed them